src/main/java/io/axoniq/axonserver/connector/event/DcbEventChannel.java

```diff
@@ -138,6 +140,24 @@ default ResultStream<StreamEventsResponse> stream(StreamEventsRequest request, i
      */
     CompletableFuture<GetTagsResponse> tagsFor(long sequence);
 
+    /**
+     * Assigns tags to the event identified by its sequence.
+     *
+     * @param sequence the sequence of an event
+     * @param tags     the tags to add to the event
+     * @return a CompletableFuture containing the response
+     */
+    CompletableFuture<AddTagsResponse> addTags(long sequence, Collection<Tag> tags);
+
+    /**
+     * Removes tags from the event identified by its sequence.
+     *
+     * @param sequence the sequence of an event
+     * @param tags     the tags to remove from the event
+     * @return a CompletableFuture containing the response
+     */
+    CompletableFuture<RemoveTagsResponse> removeTags(long sequence, Collection<Tag> tags);
```

Comment:
Nit: Can we add a `replace` on this interface, that defaults to first invoke remove and then add? It would keep the Axon Server API simple, but provide an easy step for consumers of the Axon Server Connector.

Reply:
Fair points, @m1l4n54v1c! Let's drop this idea then. 
The request mainly comes from anticipating how users will want to integrate with this.
Assuming that Axon Framework will have some integration, I'd expect people would want a `replace` functionality; hence why I was looking for it.
However, that's all in the future, no necessity for now whatsoever.